explorer/src/pages/AccountDetailsPage.tsx

```diff
@@ -195,14 +195,14 @@ export function AccountHeader({
     let token;
     let unverified = false;
 
-    if (tokenDetails) {
-      token = tokenDetails;
-    } else {
+    if (data?.nftData) {
       token = {
         logoURI: data?.nftData?.json?.image,
         name: data?.nftData?.json?.name,
       };
       unverified = true;
```

Comment:
Is this going to be a problem?

Reply:
I'm checking if `nftData` exists first and using that if it does. Otherwise fallback to token-list. What may be a problem?

Reply:
Stupid GitHub. I tried to comment on:

```
      unverified = true;
```

This is why we can't have nice things.